.github/workflows/ci.yml

```diff
@@ -89,7 +89,7 @@ jobs:
         set -ex
         ls artifacts
         cd src/cli
-        python -m pip install --upgrade pip
+        python -m pip install --upgrade pip==21.2.4
```

Comment:
```suggestion
        # Workaround: pin `pip` version until raimon49/pip-licenses#113 is fixed.
        python -m pip install --upgrade pip==21.2.4
```

Reply:
Probably want to open a work item to track undoing this change as well.